source/common/upstream/cluster_manager_impl.cc

```diff
@@ -541,11 +546,12 @@ bool ClusterManagerImpl::addOrUpdateCluster(const envoy::api::v2::Cluster& clust
   //    primary/secondary init, static/CDS init, warming all clusters, etc.
   // 2) After initial server load, we handle warming independently for each cluster in the warming
   //    map.
-  // Note: It's likely possible that all warming logic could be centralized in the init manager, but
-  //       a decision was made to split the logic given how complex the init manager already is. In
-  //       the future we may decide to undergo a refactor to unify the logic but the effort/risk to
-  //       do that right now does not seem worth it given that the logic is generally pretty clean
-  //       and easy to understand.
+  // Note: It's likely possible that all warming logic could be centralized in the init manager,
+  // but
```

Comment:
nit: strange formatting, revert? Can you potentially revert all the unrelated text formatting changes?